Automattic/woocommerce-payments
# Add payment channel information + payment identifiers to transactions

Fixes #4101 

#### Changes proposed in this Pull Request

This PR introduces two changes:
- Channel information (In-Person or Online) to the transactions list + individual transaction info. All transactions marked as `card_present` or `interac_present` are marked as `In-Person`. All others are marked as `Online`.
- Each payment method is now displayed with an identifier (last 4 digits for Cards, Bank details for others)









#### Testing instructions





* This PR should be tested with PR `2215` from the server
* Execute all steps necessary in the server's PR
* Connect an account with some in-person transactions
* Open the Transactions list view
* Make sure transactions have identifiers on payment methods (ie, last 4 digits for card...)
* Make sure there is a new column called 'Channel' and contains either `In-Person` or `Online`
* Click on a transaction and double-check if there is a mention of `In-Person` or `Online` on the individual transaction page

<img width="1580" alt="Screen Shot 2022-07-18 at 14 51 16" src="https://user-images.githubusercontent.com/12542046/179718450-32d75ac7-d61e-4e9e-90f6-7ed45f49734d.png">


<img width="1036" alt="individual_transaction" src="https://user-images.githubusercontent.com/12542046/177128573-958bbdd2-71ce-49b0-b272-cfe0af311a11.png">

-------------------


**Post merge**
